File: articles/translation-movements-and-computational-knowledge-transfer/prolog/transfer_score.pl
sum_list([], 0).
sum_list([H|T], Sum) :- sum_list(T, Rest), Sum is H + Rest.

transfer_score(Scores, Score) :-
    sum_list(Scores, Sum),
    length(Scores, N),
    Score is Sum / N.

% Query example:
% ?- transfer_score([0.98,0.90,0.94], Score).

File: articles/translation-movements-and-computational-knowledge-transfer/fortran/transfer_score.f90
program transfer_score_example
  implicit none
  real :: scores(9)
  scores = (/0.98, 0.90, 0.94, 0.88, 0.94, 0.90, 0.96, 0.84, 0.96/)
  print *, "transfer_score=", sum(scores) / size(scores)
end program transfer_score_example
